2020/01/07 09:33:24.828 - MainThread - imsaver.py.__init__.85 - WARNING
Image saver could not create directory: .\2020\January\07
 Using current directory instead
[WinError 5] Access is denied: 'X:\\Sequence Log\\2020\\January\\07'


2020/01/07 10:34:31.942 - Dummy-7 - networker.py.run.130 - ERROR
Python server: client terminated connection before message was sent. Re-inserting message at front of queue.
[WinError 10053] An established connection was aborted by the software in your host machine


2020/01/07 10:34:32.021 - Dummy-7 - networker.py.run.138 - ERROR
Python server: client terminated connection before receive.
[WinError 10053] An established connection was aborted by the software in your host machine


2020/01/07 10:36:47.392 - Dummy-8 - networker.py.run.138 - ERROR
Python server: client terminated connection before receive.
[WinError 10053] An established connection was aborted by the software in your host machine


2020/01/07 10:48:53.747 - MainThread - settingsgui.py.load_settings.507 - WARNING
Image analysis settings failed to load line from default.config:
not enough values to unpack (expected 2, got 1)


2020/01/07 11:36:50.504 - MainThread - runid.py.multirun_step.268 - WARNING

Multirun run 149 fitting failed. Histogram data in 0Meas0_1.csv


2020/01/07 14:17:18.698 - MainThread - AndorFunctions.py.__init__.48 - ERROR
Andor functions dll file not found.

Traceback (most recent call last):
  File "./andorcamera\AndorFunctions.py", line 46, in __init__
    self.dll = cdll.LoadLibrary(dllpath) # note dll path must be absolute, not relative
  File "C:\ProgramData\Anaconda3\lib\ctypes\__init__.py", line 434, in LoadLibrary
    return self._dlltype(name)
  File "C:\ProgramData\Anaconda3\lib\ctypes\__init__.py", line 356, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: [WinError 126] The specified module could not be found
2020/01/07 14:17:18.773 - MainThread - cameraHandler.py.__init__.63 - WARNING
Andor EMCCD not initialised.
'Andor' object has no attribute 'dll'


2020/01/07 14:34:27.601 - MainThread - master.py.closeEvent.401 - WARNING
camera safe shutdown failed.
'Andor' object has no attribute 'dll'


2020/01/07 18:27:25.958 - Dummy-1 - networker.py.run.136 - ERROR
Python server: client terminated connection before message was sent. Re-inserting message at front of queue.
[WinError 10053] An established connection was aborted by the software in your host machine


2020/01/07 18:27:26.000 - Dummy-1 - networker.py.run.145 - ERROR
Python server: client terminated connection before receive.
[WinError 10053] An established connection was aborted by the software in your host machine


